### PR TITLE
Add button--60 and button--70, calculate padding with control-height …

### PR DESCRIPTION
**Pull request by romypetitjean**
…and update scale of control-height

### CODE REVIEW

File: scss/controls/button.scss [hunk 1]
```diff
@@ -9,30 +9,29 @@ $button-active-color: $color-text-diap !default;
 $button-active-background-color: $color-ui-400 !default;
 $button-outset-box-shadow: 0 $button-outset 0 0 shade($button-background-color, 25%) !default;
 $button-disabled-opacity: 0.35 !default;
+$button-line-height: ($font-size-100 * $line-height-40);
```
Comment (bartvandebiezen) [inline scss/controls/button.scss]: Why force line-height to be px's?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +16,9 @@
 
 ## Features
 * Add input stepper component
+* Update get-svg-uri function to check for #000000 fill
```
Comment (bartvandebiezen) [inline CHANGELOG.md]: fill or stroke.

**Comment by romypetitjean** [comment]
Yes I agree to rename it @bartvandebiezen , but maybe do this in another pull-request as this pull-request is mainly about the button, I already added an issue to the Harbour JIRA board.

**Comment by romypetitjean** [comment]
@bartvandebiezen  I forgot that I also changed the control-heights in this pull-request, so I think it's ok to directly change the file name to sizes :)

**Comment by bartvandebiezen** [comment]
and you could also rename the variable control-height -> control-size :)

**Pull request merged**